lib/plugins/aws/provider/awsProvider.js

```diff
@@ -796,6 +811,9 @@ class AwsProvider {
             // See also https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html
             Resources: {
               type: 'object',
+              properties: {
+                $ref: '#/definitions/awsCfTransform/properties',
```

Comment:
Have you tested that it works as intended?

If I see correctly it translates to:

```javascript
Resources: {
  type: 'object', 
  properties: {
    type: 'object',
    properties: {...}
  },
  ...
}
```

And that looks invalid (note that AJV unfortunately does not notify on unrecognized properties in Schema, it simply ignores them).

I think we don't have to introduce `awsCfTransform` definition (no reusabilty use case at this point), so we can fix  it simply by putting definition inline

Reply:
It did work in my tests. In this case, because the JSON Pointer is to `awscfTransform/properties` and not just `awscfTransform`, it would translate to:

`Resources: {
  type: 'object', 
  properties: {
    type: 'object',
    properties:
      'Fn::Transform': {...}
  },
  ...
}
`

However, if there is no reusability use case I'll just put the properties in line.

Reply:
Indeed, sorry, didn't read carefully that per path right object is pointed.

Anyway we usually do not generalize until there's a use case for it. Thanks for update!